platform/mellanox/mlnx-platform-api/sonic_platform/chassis.py

```diff
@@ -44,8 +44,8 @@
 
 # magic code defnition for port number, qsfp port position of each hwsku
 # port_position_tuple = (PORT_START, QSFP_PORT_START, PORT_END, PORT_IN_BLOCK, EEPROM_OFFSET)
-hwsku_dict_port = {'ACS-MSN2700': 0, "LS-SN2700":0, 'ACS-MSN2740': 0, 'ACS-MSN2100': 1, 'ACS-MSN2410': 2, 'ACS-MSN2010': 3, 'ACS-MSN3700': 0, 'ACS-MSN3700C': 0, 'Mellanox-SN2700': 0, 'Mellanox-SN2700-D48C8': 0}
-port_position_tuple_list = [(0, 0, 31, 32, 1), (0, 0, 15, 16, 1), (0, 48, 55, 56, 1),(0, 18, 21, 22, 1)]
+hwsku_dict_port = {'ACS-MSN2700': 0, "LS-SN2700":0, 'ACS-MSN2740': 0, 'ACS-MSN2100': 1, 'ACS-MSN2410': 2, 'ACS-MSN2010': 3, 'ACS-MSN3700': 0, 'ACS-MSN3700C': 0, 'ACS-MSN3800': 4, 'Mellanox-SN2700': 0, 'Mellanox-SN2700-D48C8': 0}
```

Comment:
@stephenxs 
```
hwsku_dict_port = { 'ACS-MSN2010': 3, 'ACS-MSN2100': 1, 'ACS-MSN2410': 2, 'ACS-MSN2700': 0, 'Mellanox-SN2700': 0, 'Mellanox-SN2700-D48C8': 0, 'LS-SN2700':0, 'ACS-MSN2740': 0, 'ACS-MSN3700': 0, 'ACS-MSN3700C': 0, 'ACS-MSN3800': 4 }
```

Reply:
Hi @nazariig 
Originally I have considered this kind of refactoring and I understand the motivation.
Eventually, I gave up doing in this way because I thought it's better to put the SKUs having name convention of "ACS-MSNXXX" in the first half of the list since they are more frequently referenced. By doing so it's easier for developers to find them when reading the code or debugging.
How do you think?

Reply:
@stephenxs 

> I gave up doing in this way because I thought it's better to put the SKUs having name convention of "ACS-MSNXXX" in the first half of the list since they are more frequently referenced.

LS-SN2700 does not belong to the one what is frequently referenced

> Originally I have considered this kind of refactoring and I understand the motivation.

Let's go with the suggested way

Also please use either ' or "

Reply:
fixed.